src/current/v23.2/architecture/distribution-layer.md

```diff
@@ -89,7 +89,7 @@ gRPC is the software nodes use to communicate with one another. Because the dist
 
 gRPC requires inputs and outputs to be formatted as protocol buffers (protobufs). To leverage gRPC, CockroachDB implements a protocol-buffer-based API.
 
-By default, the network timeout for RPC connections between nodes is {{site.data.constants.cockroach_network_timeout}}, with a connection timeout of {{site.data.constants.cockroach_network_connection_timeout}}, in order to reduce unavailability and tail latencies during infrastructure outages. This can be changed via the environment variable `COCKROACH_NETWORK_TIMEOUT` which defaults to {{site.data.cockroach_network_timeout}}.
+By default, the network timeout for RPC connections between nodes is {{site.data.constants.cockroach_network_timeout}}, with a connection timeout of {{site.data.constants.cockroach_network_connection_timeout}}, in order to reduce unavailability and tail latencies during infrastructure outages. This can be changed via the environment variable `COCKROACH_NETWORK_TIMEOUT` which defaults to {{site.data.constants.cockroach_network_timeout}}.
```

Comment:
```suggestion
By default, the network timeout for RPC connections between nodes is {{site.data.constants.cockroach_network_timeout}}, with a connection timeout of {{site.data.constants.cockroach_network_connection_timeout}}, in order to reduce unavailability and tail latencies during infrastructure outages. This can be changed via the environment variable `COCKROACH_NETWORK_TIMEOUT`, which defaults to {{site.data.constants.cockroach_network_timeout}}.
```

Reply:
HOW DARE YOUUUUUUUUUUUU

ok fixed :-)